resolvedvelocities/synthetic_data/Field.py

```diff
@@ -0,0 +1,76 @@
+# Field.py
+import numpy as np
+from apexpy import Apex
+from scipy import interpolate
+import coord_convert as cc
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+
+import configparser
+
+class Field(object):
+
+    def __init__(self, config):
+
+        self.read_config(config)
+
+        # convert list from config file to arrays
+        self.field_coords = np.array(self.field_coords)
+        self.field_values = np.array(self.field_values)
+
+        # initialize Apex object
+        self.A = Apex(date=self.apex_year)
+
+        self.map_velocity_field(self.field_coords, self.field_values)
+        self.convert_to_ECEF()
+        self.create_interpolators()
+        # self.plot_ionosphere()
+
+    def read_config(self, config_file):
+
+        config = configparser.ConfigParser()
+        config.read(config_file)
+        self.__dict__.update(config.items('FIELD'))
```

Comment:
What is happening here? Can this be:

    self.config = config 

instead? Generally it isn't a good idea to use eval if one can help it. I can make a better suggestion once I know what the intent here is.